test/vendor/github.com/Microsoft/hcsshim/pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -0,0 +1,204 @@
+package securitypolicy
+
```

Comment:
There are a few files in ``test/vendor/github.com/Microsoft/hcschim/pkg/securitypolicy``. Is this supposed to be in this PR?